pdk/1.0/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -0,0 +1,41 @@
+= Compiling Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+You can compile the custom policy project after you complete all the steps in xref:policies-pdk-create-project.adoc[].
+
+You must compile the custom policy before you can deploy it to a Flex Gateway.
+
+After you compile your project, test your policy with the following tutorials:
```

Comment:
I'm thinking this and the following two lines should be a step at the end of the tutorial steps, otherwise they might not remember to look at the beginning of the page after they compile the project.